overlord/devicestate/handlers_install.go

```diff
@@ -1356,7 +1357,7 @@ func createSaveBootstrappedContainer(saveNode string) (secboot.BootstrappedConta
 		return nil, err
 	}
 
-	return secbootCreateBootstrappedContainer(secboot.DiskUnlockKey(saveEncryptionKey), saveNode), nil
+	return secbootCreateBootstrappedContainer(secboot.DiskUnlockKey(saveEncryptionKey), saveNode, nil), nil
```

Comment:
shouldn't we take the value from the cache passed it in? is the test currently only dealing with mocked stuff?

Reply:
`createSaveBootstrappedContainer` is only called during factory reset https://github.com/canonical/snapd/blob/39dc92b57ac31b4335b80f214daddc53890d2605/overlord/devicestate/handlers_install.go#L629

The value is passed from cache in `EncryptPartitions()` in the install code path.

Reply:
can you point to the relevant code?

Reply:
Ahh yes, my bad https://github.com/canonical/snapd/blob/39dc92b57ac31b4335b80f214daddc53890d2605/overlord/devicestate/handlers_install.go#L1297

Reply:
I was confused because I missed this change:
https://github.com/canonical/snapd/pull/14918/files#diff-f796a3322af4dad794f8018bb1c4de9e30562adc6e9a5a43e30595e157a138a2R722